assistants/core/package.json

```diff
@@ -12,6 +12,7 @@
     "rules only",
     "core rules",
     "sketch assistant",
+    "public",
```

Comment:
Would core be listed since it's not a 🔋-included Assistant? Was this meant to be Tidy instead?

Reply:
You are correct 🙈